docs/component-guidance/inputter.md

```diff
@@ -0,0 +1,251 @@
+
+The input size correlates to its expected value, such as postcodes, telephone numbers and max length.
+
+---
+
+### Interactive states
+
+Each `muon-inputter` component has a visual cue of its interactive state that is common or coherent across all its types.
+
+#### Text type inputs
+
+![Text type inputs](../images/inputter/inputter-text-type-inputs.png)
+
+#### Checkboxes
+
+![Checkboxes](../images/inputter/inputter-checkboxes.png)
+
+#### Radio buttons
+
+![Radio buttons](../images/inputter/inputter-radiobuttons.png)
+
+#### Select
+
+The `muon-inputter` Select opens a native dropdown - not a customised one.
+
+![Select](../images/inputter/inputter-select-states.png)
+
+#### Helper and Tip text interaction states
+
+![Helper and Tip text interaction states](../images/inputter/inputter-helperandtip.png)
+
+---
+
+### Accessibility
+
+- Focus states and tabbing order throughout the whole form. Use `muon-cta` to submit.
+- Colours and contrast meet or exceed WCAG 2.1 AA 
+- Changing of shape on interactive states - don't rely on the use of colour alone.
+- The minimum target size for interactions (44px) https://www.w3.org/WAI/WCAG21/Understanding/target-size.html
+- To make forms usable with screen readers, Labels should always be used to ask the question that the input value answers. The exception being that Labels for Checkboxes and Radios are the answers to a posed question in the parent heading.
+
+---
+
+### Content
+
+- Inputs use native autocompletes. They will resize when using that attribute.
+- Spellcheck should be disabled for names within a text input and allow for accented characters, spaces and special characters such as hyphens, commonly found in people's names.
+- Consideration should be made for the right to left read order for internationalisation (eg. Arabic etc.) which can also affect the location of functional type icons within inputters.
+- Don't use invisible elements.
+- The input size correlates to its expected value, such as postcodes, telephone numbers and max length.
+
+---
+
+### Behaviour
+
+- Keyboard focus should tab through in the same order of appearance of the inputters.
+- Text entered into an inputter that is longer than the initial size of the field will behave like the browser's default.
+- The Search inputter will display a functional 'clear' icon at the opposite end to the search icon whilst text is being input, allowing the user to clear the contents if they wish.
+- Password should be hidden by default.
+- Clicking the functional icon on the Date should open the native datepicker calendar.
+- Textarea has a grab handle allowing the user to resize to view more of the text being entered but is  also scrollable.
+- Inputters default widths fit within the smallest viewport (320px wide)
+- By default inputter components stack in one column vertically and singly.
+- Radios and checkboxes are not selected by default leaving this choice to the user.
+
+---
+
+### Best practice
+
+#### Do
```

Comment:
Extremely long list - can there be a way to break this down?

Reply:
I've tried to make it more concise but not much I think that can be taken away without losing some meaning?